layouts/partials/footer.html

```diff
@@ -1,15 +1,13 @@
-{{ $logo := printf "img/logos/%s" site.Params.logos.footer | relURL }}
-{{ $menu := site.Menus.main }}
 {{ $year := now.Year }}
 <footer class="footer has-background-image">
   <div class="container">
     <div class="columns is-variable is-8">
       <div class="column is-one-quarter">
-        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ site.Title }} footer logo">
+        <img class="is-footer-logo" src="/img/logos/{{ site.Params.logos.footer }}" alt="{{ site.Title }} footer logo">
```

Comment:
Inlining the path to the logo, IMHO is much clearer. I'd even drop the logos parameter because this is the only place where this logo is used; but I chose to keep the parameter for now.